src/common/utility.cpp

```diff
@@ -595,4 +602,12 @@ QByteArray Utility::conflictFileBaseName(const QByteArray &conflictName)
     return "";
 }
 
+QString Utility::sanitizeForFileName(const QString &name)
+{
+    static QRegularExpression regex(QStringLiteral("[\x00-\x1f\x80-\x9f/?<>\\\\:*|\"]"));
```

Comment:
```
QString invalid("?<>\\:*|...") 
... std::remove_if(..., [&](auto c) { return !c.isPrint() || invalid.contains(c); }  ) ...
```

Reply:
That would also drop compositing unicode codepoints as far as I can tell.